app/components/settings/billing-page/membership-section/status-pill.ts

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    variant: 'teal' | 'red' | 'yellow';
```

Comment:
```suggestion
    color: 'teal' | 'red' | 'yellow';
```

More obvious what it does - clearer to use names like `@size` / `@color` wherever we can and only default to `@variant` if there's no more direct term for the customization like with `LanguageLogo`, where `color: 'color'` would be weird.